src/reactable.jsx

```diff
@@ -362,7 +362,7 @@
 
                 Ths.push(
                     <Th className={sortClass} key={index} onClick={this.handleClickTh.bind(this, column)}>
-                        {column.label}
+                        {column.label || column}
```

Comment:
I'd rather make this check more explicit - put it in a `var label = ...` before the `Ths.push` call